README.md

```diff
@@ -0,0 +1,59 @@
+# Cukinia firmware validation framework
+
+Cukinia is designed to help Linux-based embedded firmware developers
+run simple validation tests on their firmware.
+
+Cukinia integrates well with embedded firmware generation frameworks
+such as Buildroot and Yocto, and can be run manually or by your
+favourite continuous integration framework.
+
+## Project objectives
+
+Cukinia works if it offers the following value:
+
+* It is very simple to use
+* It requires no dependencies other than busybox
+* It helps developers creating better software
+
+## Basic config
+
+To run Cukinia, create a configuration describing your tests, and
+invoke it.
+
+By default, cukinia reads ``/etc/cukinia/cukinia.conf`` and executes
+all executable tests in ``/etc/cukinia/tests.d/``.
+
+A cukinia config file supports the following statements:
+
+### Test statements
+
+* ``cukinia_user <username>``: Validates that user exists
+* ``cukinia_process <pname> [user]``: Validates that process runs (as user)
```

Comment:
@macmata that's what the [ ] are meant to :) as in all posix commands interactive helps